### PR TITLE
DROOLS-2821: [DMN Designer] [IE11] Missing Edit Expression type icon

### PR DESCRIPTION
**Pull request by danielzhe**
This is the same approach used in #1766

### CODE REVIEW

**Comment by manstis** [comment]
@wmedvede Would you mind reviewing too please?

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
@danielezonca FYI, in case you've not been told, our _process_ is that PRs get submitted, reviewed by our peers (usually somebody you work with and the applicable member of QE, in our case usually @jomarko). Once the PR is approved and the build _green_ you can ask a "gate keeper" (me and some others, but it varies per repository) to merge it. It is easier for somebody who creates the PR to monitor its approval and green status than a "gate keeper" monitoring all PRs.. so I'll wait to hear from you as to when these can be merged :-)

**Comment by danielzhe** [comment]
Jenkins please retest this.

File: kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/resources/StunnerCommonImageResources.java [hunk 1]
```diff
@@ -19,16 +19,14 @@
...
 import com.google.gwt.resources.client.ClientBundle;
 import com.google.gwt.resources.client.ClientBundleWithLookup;
-import com.google.gwt.resources.client.DataResource;
 import com.google.gwt.resources.client.ImageResource;
 
 public interface StunnerCommonImageResources extends ClientBundleWithLookup {
 
     StunnerCommonImageResources INSTANCE = GWT.create(StunnerCommonImageResources.class);
 
-    @Source("images/edit.svg")
```
Comment (jomarko) [inline kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/resources/StunnerCommonImageResources.java]: I think we should remove `images/edit.svg` from resources too.

Comment (danielzhe) [reply kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/resources/StunnerCommonImageResources.java]: There are others .svg files in the same folder that looks orphan (like delete.svg and form.svg). I'll remove them all to keep it clean. What do you think?

Comment (manstis) [reply kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/resources/StunnerCommonImageResources.java]: Clean-up whatever you can :-)

Comment (jomarko) [reply kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/resources/StunnerCommonImageResources.java]: +1

**Comment by danielzhe** [comment]
@manstis Please, merge.

**Pull request merged**